#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_implicit-pointer-conversion.dpatch by Lu, Chao-Ming (Tetralet) <tetralet@luna.com.tw>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: patch-file description

@DPATCH@
diff -uNr pcmanfm-0.3.0.2.orig/src/app-chooser-dialog.c pcmanfm-0.3.0.2/src/app-chooser-dialog.c
--- pcmanfm-0.3.0.2.orig/src/app-chooser-dialog.c	2006-08-14 19:09:14.000000000 +0800
+++ pcmanfm-0.3.0.2/src/app-chooser-dialog.c	2006-09-12 22:00:51.000000000 +0800
@@ -15,9 +15,11 @@
 
 #include <gtk/gtk.h>
 #include <glib.h>
+#include <string.h>
 
 #include "vfs-mime-type.h"
 #include "vfs-app-desktop.h"
+#include "glade-support.h"
 
 enum{
     COL_APP_ICON = 0,
diff -uNr pcmanfm-0.3.0.2.orig/src/ptk/ptk-bookmarks.c pcmanfm-0.3.0.2/src/ptk/ptk-bookmarks.c
--- pcmanfm-0.3.0.2.orig/src/ptk/ptk-bookmarks.c	2006-08-07 14:10:09.000000000 +0800
+++ pcmanfm-0.3.0.2/src/ptk/ptk-bookmarks.c	2006-09-12 21:59:23.000000000 +0800
@@ -13,6 +13,7 @@
 #include "ptk-bookmarks.h"
 
 #include <stdio.h>
+#include <string.h>
 
 const char bookmarks_file_name[] = ".gtk-bookmarks";
 static PtkBookmarks bookmarks = {0};
diff -uNr pcmanfm-0.3.0.2.orig/src/ptk/ptk-dir-tree-view.c pcmanfm-0.3.0.2/src/ptk/ptk-dir-tree-view.c
--- pcmanfm-0.3.0.2.orig/src/ptk/ptk-dir-tree-view.c	2006-07-27 19:46:33.000000000 +0800
+++ pcmanfm-0.3.0.2/src/ptk/ptk-dir-tree-view.c	2006-09-12 22:00:06.000000000 +0800
@@ -11,6 +11,7 @@
 */
 
 #include "ptk-dir-tree-view.h"
+#include "ptk-file-icon-renderer.h"
 
 #include <glib.h>
 #include <glib/gi18n.h>
--- pcmanfm-0.3.0.2/src/main.c~	2006-08-14 05:09:14.000000000 -0600
+++ pcmanfm-0.3.0.2/src/main.c	2006-09-25 21:48:08.000000000 -0600
@@ -26,6 +26,8 @@
 #include "glade-support.h"
 #include "settings.h"
 
+#include "app-chooser-dialog.h"
+
 static char* init_path = NULL;
 
 static int sock;
